src/App.cpp

```diff
@@ -13,7 +13,7 @@ void App::Start() {
 
 void App::Update() {
     auto input = Util::Input::GetInstance();
-    auto [x, y] = input->GetCursorPosition();
+    auto cursorPos = input->GetCursorPosition();
```

Comment:
`Input` is stateless, using a namespace (similar to a logger) could be better than a singleton class.

Reply:
https://github.com/ntut-open-source-club/practical-tools-for-simple-design/issues/33